eZ/Publish/API/Repository/RoleService.php

```diff
@@ -26,8 +27,144 @@
 interface RoleService
 {
     /**
+     * Creates a new RoleDraft.
+     *
+     * @since 6.0
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException if the authenticated user is not allowed to create a role
+     * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException if the name of the role already exists or if limitation of the
+     *                                                                        same type is repeated in the policy create struct or if
+     *                                                                        limitation is not allowed on module/function
+     * @throws \eZ\Publish\API\Repository\Exceptions\LimitationValidationException if a policy limitation in the $roleCreateStruct is not valid
+     *
+     * @param \eZ\Publish\API\Repository\Values\User\RoleCreateStruct $roleCreateStruct
+     *
+     * @return \eZ\Publish\API\Repository\Values\User\RoleDraft
+     */
+    public function createRoleDraft(RoleCreateStruct $roleCreateStruct);
```

Comment:
@lolautruche The API changes where designed by @andrerom, and I think his reason for not changing `createRole()` was BC. Awaiting his feedback before I change anything here.